internal/controller/cloudsyncmgr/cloudsync.go

```diff
@@ -65,6 +67,13 @@ func (c *CloudSyncImpl) InitiateCloudSync(isCloudSet string) (err error) {
 		if strings.Compare(strings.ToLower(isCloudSet), "true") == 0 {
 			log.Println("CloudSync init set")
 			isCloudSyncSet = true
+			secure := os.Getenv("SECURE")
+			if len(secure) > 0 {
+				if strings.Compare(strings.ToLower(secure), "true") == 0 {
+					log.Println(logPrefix, "Orchestration init with secure option")
```

Comment:
`logPrefix = "[cloudsyncmgr]"` change to `logPrefix = "[cloudsyncmgr] "`

Reply:
what is the change in logPrefix? its already cloudsyncmgr

Reply:
Done!!